src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/edit/files/main.js

```diff
@@ -337,6 +362,34 @@ define(['services/sulumedia/media-manager',
          */
         enableDropzone: function() {
             this.sandbox.emit('husky.dropzone.' + this.options.instanceName + '.enable');
+        },
+
+        /**
+         * loads the collection-data into this.data. is automatically executed before component initialization
+         * @returns {*}
+         */
+        loadComponentData: function() {
+            var promise = this.sandbox.data.deferred();
+            CollectionManager.loadOrNew(this.options.id, UserSettingsManager.getMediaLocale()).then(function(data) {
```

Comment:
Why don't we just return the promise from `loadOrNew`, but create a new one instead?